app/graphql/mutations/user_count/create.rb

```diff
@@ -0,0 +1,16 @@
+module Mutations
+  module UserCount
+    class Create < Mutations::BaseMutation
```

Comment:
@kanth5 - can you clarify the goal of this mutation?

Reply:
This is supposed to contain various counters for a user and can be expanded in future to allow more scenarios. We have used it to keep a count of 2 scenarios.
1) Profanity in a response
2) Short response that is less than 50 characters.

Reply:
We would leave it to Commutatus to decide on the naming convention and whether the calculations for the counters should happen on the backend and have not made any further changes in this regard.